compile_flags.txt

```diff
@@ -3,8 +3,8 @@
 -xc++
 -nostdinc
 -Ibazel-bin/external/dawn/include
--Ibazel-bin/external/ada-url/_virtual_includes/ada-url/
--Ibazek-bin/external/simdutf/virtual_includes/simdutf/
+-Ibazel-bin/external/ada-url/
```

Comment:
I don't have headers listed in either of these paths – using just `external/simdutf/` and `external/ada-url/` instead seems to work instead. (In some cases we need the bazel-bin/ prefix because there are generated headers but here this is not the case.)

Reply:
Since, we already have `Iexternal/ada-url` and simdutf, I'll just remove these lines then. Thanks for the review @fhanau